tests/test_recorders.py

```diff
@@ -531,24 +531,21 @@ def test_concatenated_dataframes(simple_storage_model):
 
 
 @pytest.mark.parametrize("complib", [None, "gzip", "bz2"])
-def test_csv_recorder(simple_linear_model, tmpdir, complib):
+def test_csv_recorder(simple_storage_model, tmpdir, complib):
+    """Test the CSV Recorder
     """
-    Test the CSV Recorder
-
-    """
-    model = simple_linear_model
+    model = simple_storage_model
     otpt = model.nodes['Output']
-    model.nodes['Input'].max_flow = 10.0
     otpt.cost = -2.0
 
     # Rename output to a unicode character to check encoding to files
     if sys.version_info.major >= 3:
```

Comment:
Check not needed now python 2 support has been dropped?

Reply:
Could spot. I'll remove this too.